pkg/storaged/storage.css

```diff
@@ -641,3 +641,71 @@ td.storage-action {
     opacity: 0;
     visibility: hidden;
 }
+
+.modal-body .intermission {
+    margin-top: 1.5ex;
```

Comment:
Yeah, `ex` is for vertical distances, and `em` for horizontal ones.

I have to admit that not much thought has gone into the CSS.  "Whatever looks about right"...

Reply:
@Gundersanne: There are a bunch of different units in CSS. Each has their own use.

Relative units:

- **%**: relative to current element
- **em**: relative to the font size of the element (so the meaning could vary based on the element it is applied to)
- **ex**: also relative to the element, but based on the specific x-height of the font (that is, relative to the height of the 'x' character) — this is often, but not always, 0.5 em
- **rem**: same as em, but it's based on the root element (basically what's defined for `<body>`), not the current element, so you can be sure 1rem is the same everywhere
- **ch**: similar to ex, but based on the width of the `0` instead of the height of the `x`... this can be especially useful for monospaced text, as 80ch in monospaced text means exactly 80 characters wide. For proportional text, around 75% of ch (give or take, depending on the font and letters used in the content) is about right. That is, for porportional fonts, if you want 80 characters wide, you'd probably specify something like 60ch. [See Eric Meyer's blog post about this for more.](https://meyerweb.com/eric/thoughts/2018/06/28/what-is-the-css-ch-unit/)

Absolute units:
- **px**: it's often thought to be the device's pixels, but this isn't always the case for zoomed text, HiDPI screens, mobile devices, and so on
- **in**/**cm**/**mm**/etc.: these are exactly what they sound like and are sizes for physical distances... they're not used much on the web, especially in Cockpit

Screen units:
- **vw**: viewport width — 1% of the width of the page. 100vw == the visible page's width; 50vw == half the visible page's width
- **vh**: viewport height, same as vw, but for height; 100vh == the complete visible height of the page
- **vmin**: viewport minimum; the smallest of vw or vh... useful if you don't know if the screen is portrait or landscape
- **vmax**: viewport maximum; the largest of vw or vh…. also useful if rotation of screen isn't known

...and you can mix and match all the above units by tossing them into a `calc()`, like: `calc(10vw + 3rem - 12px)` (this is a convoluted example, but it's: 10% of the screen plus 3 lines high of the base font size minus 12 pixels — and the browser would figure this out and give you the size you want).

These units can be used for width, height, line-height, min-width, min-height, max-width, max-height, font-size, border-width, and so on.

---
There's a nice overview of these (and other) units at https://webdesign.tutsplus.com/articles/7-css-units-you-might-not-know-about--cms-22573

Reply:
> by tossing them into a calc()

Oh, neat.  I take it that we can use this in Cockpit without reservation, right?